bin/init.bash

```diff
@@ -117,32 +117,32 @@ set_storage_class() {
           storage_class=cinder-storage
           es_storage_class=cinder-storage
 
-          [ "$(yq read "$file" 'storageClasses.cinder.enabled')" = "null" ] &&
+          [ "$(yq read "$file" 'storageClasses.cinder.enabled')" = "false" ] &&
```

Comment:
Isn't it better to have this like:
```suggestion
          [ "$(yq read "$file" 'storageClasses.cinder.enabled')" != "true" ] &&
```
Then it doesn't matter what it's set to from the beginning. As long as it's not `true`, change it to `true`

Reply:
Yes probably. Makes it a bit more robust.